crates/bevy_math/src/rects/irect.rs

```diff
@@ -19,6 +19,12 @@ pub struct IRect {
 }
 
 impl IRect {
+    /// An empty `IRect`, represented by maximum and minimum corner points
+    /// with all `i32::MAX` values.
+    pub const EMPTY: Self = Self {
+        max: IVec2::MAX,
+        min: IVec2::MAX,
+    };
```

Comment:
Why use the maximum value? The docs seem to indicate that negative or zero sizes will return this empty rect, but a user might assume that in those cases its size would be limited to zero, not jump to the max value.

Some users may be storing positional data and stuff in these rects, meaning that jumping to the max value could be an issue.

Reply:
FWIW the behavior of the method was not changed, so it's not that we're explicitly returning these constants. It's a good point though, I just followed the requirements from the original issue. Might be good to open a new one if this is something we should address later, maybe?

Reply:
Great, I like that convention. We should change to that and clearly document it.

Reply:
I agree with @mnmaita this might be better saved for a followup PR. Not sure we want to change these kinds of semantics in what's supposed to be a rename PR.